app/models/spree/admin/tabs/user_default_tabs_builder.rb

```diff
@@ -0,0 +1,116 @@
+module Spree
+  module Admin
+    module Tabs
+      class UserDefaultTabsBuilder
+        include Spree::Core::Engine.routes.url_helpers
+
+        def build
+          root = Root.new
+          add_account_tab(root)
+          add_addresses_tab(root)
+          add_orders_tab(root)
+          add_items_tab(root)
+          add_store_credits_tab(root)
+          root
+        end
+
+        private
+
+        def add_account_tab(root)
+          tab =
+            TabBuilder.new(account_config).
+            with_active_check.
+            build
+
+          root.add(tab)
+        end
+
+        def account_config
+          {
+            icon_name: 'person.svg',
+            name: 'admin.user.account',
+            url: ->(resource) { edit_admin_user_path(resource) },
+            classes: 'nav-link',
+            partial_name: :account
+          }
+        end
+
+        def add_addresses_tab(root)
+          tab =
+            TabBuilder.new(addresses_config).
+            with_active_check.
+            build
+
+          root.add(tab)
+        end
+
+        def addresses_config
+          {
+            icon_name: 'pin-map.svg',
+            name: 'admin.user.addresses',
+            url: ->(resource) { addresses_admin_user_path(resource) },
+            classes: 'nav-link',
```

Comment:
Just a question - is there any scenario where we wouldn't use the `nav-link` class?

Reply:
I think the template code should ideally manage which classes are used, based on other factors (e.g. the configuration). I wouldn't add a classes element here, as it leaks the abstraction a little bit

Reply:
It indeed doesn't seem so. But do you mean deleting it from TabBuilder's config, and embedding it in the config that’s passed to Tab's initializer? This would mean we don’t allow for customization but if it’s not a desired feature, then I’m good with skipping it 🙂

Reply:
Done.